strings/strings.json

```diff
@@ -2707,6 +2707,14 @@
     "en": "Show more results for the {0} category",
     "fr": "Afficher plus de résultats pour la catégorie {0}"
   },
+  "ShowLessHierarchicalResults": {
+    "en": "Show fewer results for the {0} facet",
+    "fr": "Afficher moins de résultats pour la facette {0}"
+  },
+  "ShowMoreHierarchicalResults": {
+    "en": "Show more results for the {0} facet",
+    "fr": "Afficher plus de résultats pour la facette {0}"
+  },
```

Comment:
When no value is selected, "ShowMoreFacetResults" is used instead, so I made sure the locale is the same for both.